Makefile

```diff
@@ -1,3 +1,14 @@
+DESTDIR       =
```

Comment:
Shouldn't it be:
DESTDIR?=
PREFIX=$(DESTDIR)/usr/local

Reply:
This will still override `DESTDIR` with empty string.

Reply:
Fixed that.